src/ArgSortMsg.chpl

```diff
@@ -27,6 +27,15 @@ module ArgSortMsg
     use SegmentedArray;
     use Reflection;
     use Errors;
+    use Logging;
+
+    const asLogger = new Logger();
+
+    if v {
+        asLogger.level = LogLevel.DEBUG;
+    } else {
+        asLogger.level = LogLevel.INFO;
+    }
```

Comment:
I wonder if instead of setting the level after creation, if this should just pass the level to the Logger. Something like:

```chpl
const asLogger = new Logger(level=if v then LogLevel.DEBUG else LogLevel.INFO);
```

Or if this logic is the same for all loggers, it could be moved into the Logger type instead of being at all the call sites.

I was also wondering if there needs to be a logger per module, or if you could just create a single global logger in Logging.chpl and have these modules use it. e.g. instead of creating `asLogger` here, just use a global `logger`